### ISSUE TITLE
"red green" bars are missing in the new Flows screen

### ISSUE THREAD

**Issue by yaronlevi** [PrefectHQ/prefect#9485]
### First check

- [X] I added a descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the Prefect documentation for this issue.
- [X] I checked that this issue is related to Prefect and not one of its dependencies.

### Bug summary

In the new Flows screens, the "red green" bars that represent the frequency of all the Flows/Deplyments is gone.

![image](https://user-images.githubusercontent.com/5398977/236885846-a242effa-25ca-4ce7-bc49-65532f26b641.png)

This is a super useful view since it allows in one quick page scroll to make sure all Flows are running as needed.
There is no other way in Prefect's UI to get such information in such a quick way.

### Reproduction

```python3
Happens in the Prefect Cloud UI in the new Flows screen.
```

### Versions

```Text
Prefect cloud.
```

**Comment by OptimeeringBigya** [comment]
Just for more information, The new UI mentioned is from v2.10.7 changes where deployments pages was merged into the flow pages.

Additionally, coming from v1, I have noticed that the flow pages `[<UI-URL>//flows/flow/<flow_id>]` no longer has graphs showing last runs as well. I am not very fond of reading the dates on each flow run.

**Comment by yaronlevi** [comment]
Just to give more context.

Let's say I have a flow that runs once a month.
The "red green" bars allowed me to see that indeed it run each month.
If it didn't run for a specific month, It would be very clear as there will be a gap and a bar missing.

Now, think it's not just one flow that runs once a month, but tens of flows like this.
The "red green" bars page would allow in a quick glance to scroll and look for those gaps (bars missing).

**Comment by yaronlevi** [comment]
@znicholasbrown Hi Nicholas, any idea if the "red green" bars will be brought back to the UI somewhere? Those were crucial for our day-to-day monitoring.

**Comment by yaronlevi** [comment]
Thank you!!!
Resolved in 2.10.12
